crates/relayer-framework/src/base/one_for_all/instances/packet_relayers.rs

```diff
@@ -4,17 +4,18 @@ use crate::base::one_for_all::traits::relay::OfaBaseRelay;
 use crate::base::one_for_all::traits::relay::OfaRelayPreset;
 use crate::base::relay::impls::packet_relayers::ack::base_ack_packet::BaseAckPacketRelayer;
 use crate::base::relay::impls::packet_relayers::general::full_relay::FullRelayer;
-use crate::base::relay::impls::packet_relayers::general::retry::RetryRelayer;
 use crate::base::relay::impls::packet_relayers::receive::base_receive_packet::BaseReceivePacketRelayer;
 use crate::base::relay::impls::packet_relayers::receive::skip_received_packet::SkipReceivedPacketRelayer;
 use crate::base::relay::traits::packet_relayer::PacketRelayer;
 use crate::base::relay::traits::packet_relayers::ack_packet::AckPacketRelayer;
 use crate::base::relay::traits::packet_relayers::receive_packet::ReceivePacketRelayer;
 use crate::common::one_for_all::types::relay::OfaRelayWrapper;
+use crate::full::one_for_all::traits::relay::OfaFullRelay;
+use crate::full::relay::impls::packet_relayers::retry::RetryRelayer;
 
 pub fn full_packet_relayer<Relay>() -> PhantomData<impl PacketRelayer<OfaRelayWrapper<Relay>>>
 where
-    Relay: OfaBaseRelay,
+    Relay: OfaFullRelay,
```

Comment:
We can replace the body with `PhantomData::<FullRelayer>` and keep `OfaBaseRelay` here.